### PR TITLE
Add stop project function

### PR DESCRIPTION
**Pull request by JiepengTan**
Add stop project function:
```js
window.stopProject = async () => {
   ...
}
```

### CODE REVIEW

File: cmd/spx/template/game.js [hunk 1]
```diff
@@ -118,14 +128,37 @@ class GameApp {
...
 
+    async stopProject(resolve, reject) {
+        console.log("stop project")
+        if (this.editor == null) {
+            resolve()
+            return
+        }
+        this.stopGameTask++
+        await this.stopGame(() => {
+            this.isEditor = true
```
Comment (nighca) [inline cmd/spx/template/game.js:150]: What does `isEditor` mean? 

The current logic indicates it is equivalent to `!isRunningGame`. Is that expected?

Comment (JiepengTan) [reply cmd/spx/template/game.js:150]: This is a tag used to indicate whether the focus should be on the game canvas or the editor canvas.

Comment (nighca) [reply cmd/spx/template/game.js:150]: When should I call `StopProject` in Builder?

Comment (JiepengTan) [reply cmd/spx/template/game.js:150]: When the user wants to switch to another project

Comment (nighca) [reply cmd/spx/template/game.js:150]: OK

**Review by nighca — APPROVED** [review]

**Pull request merged**